.github/workflows/ci.yml

```
@@ -0,0 +1,73 @@
+
+jobs:
+  components:
+    name: Components Library
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./components
+
+    steps:
+      - name: 코드 가져오기
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)